ffmpeg/_utils.py

```diff
@@ -1,31 +1,9 @@
 from __future__ import unicode_literals
 from builtins import str
-from past.builtins import basestring
 import hashlib
 import sys
 
 
-if sys.version_info.major == 2:
-    # noinspection PyUnresolvedReferences,PyShadowingBuiltins
-    str = str
-
-try:
-    from collections.abc import Iterable
-except ImportError:
-    from collections import Iterable
```

Comment:
Did you check whether this runs?

Reply:
Good catch, this should stay here. I'm gonna push the commit again.